search/templatetags/search.py

```diff
@@ -92,6 +92,8 @@ def display_facet(context, flt, facet, facet_type, title=""):
             context['sort'] if context['sort'] is not None else '',
             context['weights'] or ''
         )
+        if context['similar_to'] is not None:
+            element['add_filter_url'] += '&similar_to={}'.format(context['similar_to'])
```

Comment:
Yes I agree, I did not update stuff here because this will be part of the refactoring of the whole search views/template code that I've been planning to do. All URL that we build (filter urls, pagination, etc.) should be built by some object that knows how to handle this and then that object should do the proper encoding.